stellar/deploy-contract.js

```diff
@@ -332,11 +274,28 @@ function main() {
     addBaseOptions(deployCmd, { address: true });
 
     // 3rd level commands for `deploy`
-    const deployContractCmds = Array.from(SUPPORTED_CONTRACTS).map((contractName) => {
+    const deployContractCmds = Array.from(SUPPORTED_STELLAR_CONTRACTS).map((contractName) => {
         const command = new Command(contractName)
             .description(`Deploy ${contractName} contract`)
-            .addOption(new Option('--wasm-path <wasmPath>', 'path to the WASM file'))
-            .addOption(new Option('--version <version>', 'version of the contract to deploy (e.g., v1.0.0)'))
+            .addOption(
+                new Option('--wasm-path <wasmPath>', 'Path to the WASM file (required if --version is not used)').conflicts('version'),
```

Comment:
```suggestion
                new Option('--artifactPath <artifactPath>', 'Path to the contract WASM artifact (required if --version is not used)').conflicts('version'),
```
standardize the flags

Reply:
Sure, you'll need to update the axelarate repo as well for the orchestration to keep working when scripts change

Reply:
done